snakesist/exist_client.py

```diff
@@ -382,17 +382,24 @@ def query(self, query_expression: str) -> etree._Element:
         :return: The query result as a ``delb.Document`` object.
         """
 
-        params = {
-            "_howmany": "0",
-            "_indent": "no",
-            "_wrap": "yes",
-            "_query": query_expression,
-        }
-
-        response = requests.get(
+        request_body = (
+            '<query '
+            f'xmlns="{EXISTDB_NAMESPACE}" '
+            'start="1" '
+            'max="0" '
+            'cache="no">'
+            f'<text> <![CDATA[{query_expression}]]></text>'
```

Comment:
a test should cover that documents that contain CDATA chunks are processed properly.